src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx

```diff
@@ -146,7 +146,7 @@ export default function VideoVisitLocation({ header, appointment, facility }) {
                 {phone && (
                   <>
                     <br />
-                    <FacilityPhone contact={phone} />
+                    <FacilityPhone contact={phone} level={3} />
```

Comment:
Was there an axe failure on this page without adding this prop?

Reply:
No, just noticed a couple console errors while testing and the text looked off without the proper heading level.